src/chronometer.js

```diff
@@ -1,33 +1,62 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+      this.currentTime = 0
+      this.intervalId = null
   }
 
   start(printTimeCallback) {
     // ... your code goes here
+   this.intervalId = setInterval(() => {
+        this.currentTime++
+        if (printTimeCallback) printTimeCallback()
+    }, 1000);
   }
 
+
   getMinutes() {
     // ... your code goes here
+    if (this.currentTime === 0 ) {
```

Comment:
There's no need for this condition here. If currentTime is zero, 0 / 60 = 0 so it will already return 0